Makefile

```diff
@@ -1187,6 +1189,24 @@ tags0:
 format:
 	build_tools/format-diff.sh
 
+check-format:
+	build_tools/format-diff.sh -c
+
+buck-targets:
+	python buckifier/buckify_rocksdb.py
+
+check-buck-targets: buck-targets
+	$(eval TMP=$(shell bash -c "git diff TARGETS | head -n 1"))
+	@echo Running git diff on TARGETS, and the first line of output is...
+	@echo $(TMP)
+	@if [ -z "$(TMP)" ]; then \
+		exit 0; \
+	else \
+		echo "Please run 'make buck-targets' to update TARGETS file."; \
+		echo "Do not manually update TARGETS file."; \
```

Comment:
We should tweak this either (a) to reflect the fact that the check just updated TARGETS (the user doesn't need to do it again), or (b) restore the user's file state by restoring TARGETS~ to TARGETS (let the user run "make buck-targets").

Reply:
I ended up with (b)